scripts/config-dhcp.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+#Check if isc-dhcp-server exists, if not install it.
+while true
```

Comment:
Why this `while true` is necessary ?
Can you just run the installation step if not installed ? Why dpkg `--configure -a` is here ?

Reply:
I added while true because even if i run it again, what if the same problem occurs that occurred the first time. then we have no way to check. so it will be in the loop until it verifies for sure that it is installed. and then only it will continue with the script.